AWS/mixed-environment-behind-bastion/README.md

```diff
@@ -24,12 +24,21 @@ See also: https://www.scaleft.com/docs/client/
 
 ## Install Terraform
 
-*Note that the following example has been tested up to Terraform v0.11. v0.12 introduces some breaking changes which have yet to be updated.*
-
 Download and install Terraform for your local OS here: https://www.terraform.io/downloads.html
 
 To easily switch between Terraform versions during testing, use tfenv: https://github.com/tfutils/tfenv
 
+## Generate AWS deployment RSA key.
```

Comment:
Nice, I added something for this this morning as well (but I did not add a Windows command so you win)